freqtrade/arguments.py

```diff
@@ -106,7 +106,8 @@ def __init__(self, *args, **kwargs):
     "max_open_trades": Arg(
         '--max_open_trades',
         help='Specify max_open_trades to use.',
-        type=int,
+        type=check_int_positive,
```

Comment:
`max_open_trades` can (should / must) be set to -1 for edge, so i think check_int_positive is wrong for this argument (yes, you would normally set this in the configuration anyway - but - ...)

Reply:
Ah, year, I forgot about this, thanks...
Done, returned it back to int.